### ISSUE TITLE
kube-scheduler crashing on v1.4.0-beta.8

### ISSUE THREAD

**Issue by owenhaynes** [kubernetes/kops#463]
I know v1.4.0 is in beta(out soon i hope) just wanted to have a play with some of the new features, does not look to be a kurbernetes error, from what i can tell more of a configuration error.

kube-scheduler is crashing on the following.

```
5 client_config.go:404] error creating inClusterConfig, falling back to default config: open /var/run/secrets/kubernetes.io/serviceaccount/token: no such file or directory
5 server.go:79] unable to build config from flags: invalid configuration: no configuration has been provided
```

This was a brand new AWS cluster.

**Comment by owenhaynes** [comment]
adding --master=127.0.0.1:8080 to kube-scheduler seems to have fixed it

**Comment by justinsb** [comment]
Thanks for the report @owenhaynes - I'll take a look!

**Comment by srjturner** [comment]
I know this issue is meant to be fixed, but I'm using v1.4.1 and saw the same problem with a new cluster in AWS, which I fixed with the same change to kube-scheduler (i.e. adding --master=127.0.0.1:8080)
